packages/orchestrator-ui-components/src/components/WfoSettingsPage/WfoStatus.tsx

```diff
@@ -38,7 +42,7 @@ export const WfoStatus: FC<WfoStatusProps> = ({ engineStatus }) => {
                 </EuiFlexItem>
                 <EuiFlexItem grow={false}>
                     <EuiText size="s">
-                        <p>0</p>
+                        <p>{runningProcesses || 0}</p>
```

Comment:
maybe fallbak to `-`. Since 0 has an actual meaning, it might give the user a wrong impression in case of errors

Reply:
Done